File: python/pp_sdk_integ.py
#! /usr/bin/env python
# coding: utf-8

import pprint
import datetime
import paypayopa

def createCode(client):

  now = datetime.datetime.now()
  current_time =  now.strftime('%Y%m%d%H%M%S')
  print(current_time)

  request = {
    "merchantPaymentId": "tsuk-qr-{}".format(current_time),
    "codeType": "ORDER_QR",
    "redirectUrl":"https://paypay.ne.jp",
    "redirectType":"WEB_LINK",
    "orderDescription":"tsuk shop",
    "orderItems": [
        {
            "name": "tsuk item",
            "quantity": 1,
            "unitPrice": {
                "amount": 100,
                "currency": "JPY"
            }
        }
    ],
    "amount": {
      "amount": 100,
      "currency": "JPY"
    }
  }

  # Calling the method to create a qr code
  response = client.code.create_qr_code(request)

  # Printing if the method call was SUCCESS
  pprint.pprint(response)


if __name__ == "__main__":

  API_KEY = ""
  API_SECRET = ""
  MERCHANT_ID = ""

  #Set True for Production Environment. By Default this is set False for Sandbox Environment.
  client = paypayopa.Client(auth=(API_KEY, API_SECRET), production_mode=False)
  client.set_assume_merchant(MERCHANT_ID)

  createCode(client)

File: php/pp_sdk_integ.php
<?php

require 'vendor/autoload.php';

use PayPay\OpenPaymentAPI\Client;
use PayPay\OpenPaymentAPI\Models\CreateQrCodePayload;
use PayPay\OpenPaymentAPI\Models\OrderItem;

$APIKEY = "";
$APISECRET = "";
$MID = "";

//Set True for Production Environment. By Default this is set False for Sandbox Environment.
$client = new Client([
  'API_KEY' => $APIKEY,
  'API_SECRET'=> $APISECRET,
  'MERCHANT_ID'=> $MID
],false);

// setup payment object
$CQCPayload = new CreateQrCodePayload();

// Set merchant pay identifier
$merchantPaymentId = "tsuk-qr-".time();
$CQCPayload->setMerchantPaymentId($merchantPaymentId);

// Log time of request
$CQCPayload->setRequestedAt();

// Indicate you want QR Code
$CQCPayload->setCodeType("ORDER_QR");

// Provide order details for invoicing
$OrderItems = [];
$OrderItems[] = (new OrderItem())
    ->setName('tsuk shop')
    ->setQuantity(1)
    ->setUnitPrice(['amount' => 100, 'currency' => 'JPY']);

$CQCPayload->setOrderItems($OrderItems);

// Save Cart totals
$amount = [
    "amount" => 100,
    "currency" => "JPY"
];

$CQCPayload->setAmount($amount);

// Configure redirects
$CQCPayload->setRedirectType('WEB_LINK');
$CQCPayload->setRedirectUrl('https//paypay.ne.jp');

// Get data for QR code
$response = $client->code->createQRCode($CQCPayload);

$data = $response['data'];

var_dump($data);

?>